src/components/Button/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import './button.css'
+
+export interface ButtonProps {
```

Comment:
exporting this to a different file will be much better

Reply:
something like ButtonProps.ts

Reply:
Ok